### PR TITLE
[backport/v1.18] ci: Fix broken tests

### PR DESCRIPTION
**Pull request by phlax**
Commit Message:
    
- partial fix for macOS (VM)
- fix for windows VM
- fix cache verify example
- fix VRP verify example

Additional Description:
Risk Level:
Testing:
Docs Changes:
Release Notes:
Platform Specific Features:
[Optional Runtime guard:]
[Optional Fixes #Issue]
[Optional Fixes commit #PR or SHA]
[Optional Deprecated:]
[Optional [API Considerations](https://github.com/envoyproxy/envoy/blob/main/api/review_checklist.md):]

### CODE REVIEW

**Comment by phlax** [comment]
macos is still failing - not sure where logs go

```console
================================================================================
FAIL: //test/integration:tcp_tunneling_integration_test (see /private/var/tmp/_bazel_runner/3d1a7f52fd47fc9a8db71770aacd13e5/execroot/envoy/bazel-out/darwin-fastbuild/testlogs/test/integration/tcp_tunneling_integration_test/test.log)

FAILED: //test/integration:tcp_tunneling_integration_test (Summary)
      /private/var/tmp/_bazel_runner/3d1a7f52fd47fc9a8db71770aacd13e5/execroot/envoy/bazel-out/darwin-fastbuild/testlogs/test/integration/tcp_tunneling_integration_test/test.log
      /private/var/tmp/_bazel_runner/3d1a7f52fd47fc9a8db71770aacd13e5/execroot/envoy/bazel-out/darwin-fastbuild/testlogs/test/integration/tcp_tunneling_integration_test/test_attempts/attempt_1.log

```

**Comment by phlax** [comment]
any clues on the cause/fix @zuercher ?

File: ci/docker_ci.sh [hunk 1]
```diff
@@ -141,7 +141,7 @@ for BUILD_TYPE in "${BUILD_TYPES[@]}"; do
     build_images "${BUILD_TYPE}" "$image_tag"
 
     if ! is_windows; then
-        if [[ "$BUILD_TYPE" == "" || "$BUILD_TYPE" == "-alpine" ]]; then
+        if [[ "$BUILD_TYPE" == "" || "$BUILD_TYPE" == "-alpine" || "$BUILD_TYPE" = "-google-vrp" ]]; then
```
Comment (phlax) [inline ci/docker_ci.sh:144]: s/=/==/

**Comment by phlax** [comment]
making this WIP while i try and figure the mac failure

**Comment by phlax** [comment]
great, that fixed it - ill tidy up the PR and unWIP

**Review by zuercher — APPROVED** [review]

**Pull request merged**